test/sbvr/test.coffee

```diff
@@ -28,6 +28,24 @@ module.exports = exports = (builtInVocab = false) ->
 				return
 			expectation(result)
 
+	runMigration = (describe, src, dst, expectation) ->
+		it 'Migration: \n\nVocabulary: src\n' + src + '\n\nVocabulary: dst\n' + dst, ->
+			try
+				SBVRParser.reset()
+				srcLf = SBVRParser.matchAll(src, 'Process')
+				srcSchema = LF2AbstractSQLTranslator(srcLf, 'Process')
+
+				SBVRParser.reset()
+				dstLf = SBVRParser.matchAll(dst, 'Process')
+				dstSchema = LF2AbstractSQLTranslator(dstLf, 'Process')
+
+				migration = AbstractSQLCompiler.postgres.diffSchemas(srcSchema, dstSchema)
+				console.log('GOT MIGRATION', migration)
+			catch e
+				throw e
```

Comment:
You might as well drop the `try/catch` altogether if you're just going to rethrow :P